### ISSUE TITLE
Canvas forced to be square means it can be clipped at the sides

### ISSUE THREAD

**Issue by rcoundon** [amroessam/vgauge#4]
There is currently only a height prop which means that the canvas that holds the gauge is always square and can therefore be clipped at the edges

![image](https://user-images.githubusercontent.com/9136570/72605063-fa9d5c00-3913-11ea-9f76-2d307be912a7.png)

**Comment by amroessam** [comment]
Merged and published to NPM
36de719

**Comment by rcoundon** [comment]
Brilliant, thank-you

**Comment by Emfr2582** [comment]
I'd like to second this, as I was using this as of today. 
I got the version from npm 2 days ago, and now and when I inspect the canvas element, the width and height are set twice. 

In my solution I used the default size, inside Vuetify row and col-tags

![canvas tag](https://user-images.githubusercontent.com/44642317/77520796-06f8e480-6e82-11ea-8ab5-69cc9275b09b.png)

I'd like to change the <canvas> so that the lables are clearly visable, and not clipped outside of the image.

![bensinmätare](https://user-images.githubusercontent.com/44642317/77521593-3bb96b80-6e83-11ea-92e2-3926f605ff50.png)
